Doc/library/typing.rst

```diff
@@ -3065,8 +3070,7 @@ Introspection helpers
       >>> class P(Protocol):
       ...     def a(self) -> str: ...
       ...     b: int
-      >>> get_protocol_members(P)
-      frozenset({'a', 'b'})
+      >>> assert get_protocol_members(P) == frozenset({'a', 'b'})
```

Comment:
This doctest isn't run in CI currently because it's missing the `.. doctest::` marker immediately above it. We should just add that marker so that it's run via the existing workflow.